### PR TITLE
Version bump for amazon-chime-sdk-js@3.0.0-beta.0

### PR DESCRIPTION
**Pull request by ltrung**
Bumping for the next development version.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by xuesichao — APPROVED** [review]

**Review by LeviSklaroff — APPROVED** [review]

File: src/versioning/Versioning.ts [hunk 1]
```diff
@@ -47,13 +52,14 @@ export default class Versioning {
...
     const v = VERSION.semverString.match(
-      /^(?<major>[0-9]+)\.(?<minor>[0-9]+)(?:\.(?<patch>[0-9]+))?/
+      /^(?<major>[0-9]+)\.(?<minor>[0-9]+)((?:\.(?<patch>[0-9]+))(?:-(?<pre_release>[a-zA-Z]+(\.[0-9])*))?)?/
     );
 
     return {
       major: v?.groups?.major,
       minor: v?.groups?.minor,
       patch: v?.groups?.patch,
+      pre_release: v?.groups?.pre_release,
```
Comment (xuesichao) [inline src/versioning/Versioning.ts:62]: Just curious, why we use snake case here? why not `preRelease`?

File: test/versioning/Versioning.test.ts [hunk 2]
```diff
@@ -12,9 +12,23 @@ const SHORT_SHA_RE = /^[a-f0-9]{5,40}$/;
...
+  VERSION_CORE.source +
+    '(' +
+    PRE_RELEASE.source +
+    '|' +
+    BUILD.source +
+    '|' +
+    PRE_RELEASE.source +
+    BUILD.source +
+    ')?$'
+);
```
Comment (xuesichao) [inline test/versioning/Versioning.test.ts:31]: nit: spacing
```
const VERSION_RE = new RegExp(
  VERSION_CORE.source +
  '(' +
  PRE_RELEASE.source +
  '|' +
  BUILD.source +
  '|' +
  PRE_RELEASE.source +
  BUILD.source +
  ')?$'
);
```

Comment (ltrung) [reply test/versioning/Versioning.test.ts]: This is actually from the linter.

File: src/versioning/Versioning.ts (hunk 1 above)
Comment (ltrung) [reply src/versioning/Versioning.ts]: oops. Just me with ruby style naming :)). Fixed.

**Review by xuesichao — APPROVED** [review]

**Review by xuesichao — APPROVED** [review]

**Pull request merged**